Intersect.Client/Interface/Game/DescriptionWindows/ItemDescriptionWindow.cs

```diff
@@ -354,19 +354,21 @@ protected void SetupEquipmentInfo()
                 else
                 {
                     _ = mItem.TryGetRangeFor((Stat)i, out var range);
-                    var statLow = mItem.StatsGiven[i] + range.LowRange;
-                    var statHigh = mItem.StatsGiven[i] + range.HighRange;
-
-                    if (mItem.PercentageStatsGiven[i] != 0)
-                    {
-                        rows.AddKeyValueRow(Strings.ItemDescription.StatCounts[i], Strings.ItemDescription.RegularAndPercentage.ToString(Strings.ItemDescription.StatGrowthRange.ToString(statLow, statHigh), mItem.PercentageStatsGiven[i]));
-                    }
-                    else
+                    if (range != null)
                     {
-                        rows.AddKeyValueRow(Strings.ItemDescription.StatCounts[i], Strings.ItemDescription.StatGrowthRange.ToString(statLow, statHigh));
+                        var statLow = mItem.StatsGiven[i] + range.LowRange;
+                        var statHigh = mItem.StatsGiven[i] + range.HighRange;
+
+                        if (mItem.PercentageStatsGiven[i] != 0)
+                        {
+                            rows.AddKeyValueRow(Strings.ItemDescription.StatCounts[i], Strings.ItemDescription.RegularAndPercentage.ToString(Strings.ItemDescription.StatGrowthRange.ToString(statLow, statHigh), mItem.PercentageStatsGiven[i]));
+                        }
+                        else
+                        {
+                            rows.AddKeyValueRow(Strings.ItemDescription.StatCounts[i], Strings.ItemDescription.StatGrowthRange.ToString(statLow, statHigh));
```

Comment:
Use the if statement to set the message, only call `rows.AddKeyValueRow(Strings.ItemDescription.StatCounts[i], ...)` once